transform.js

```diff
@@ -0,0 +1,5 @@
+const configify = require('./lib/configify')
+module.exports = function transform (filename, options) {
```

Comment:
this is missing from the README, your app wouldn't work without it!

Reply:
😅 added a piece for it 🙏